src/mol-geo/geometry/base.ts

```diff
@@ -33,6 +34,40 @@ export const VisualQualityOptions = PD.arrayToOptions(VisualQualityNames);
 
 //
 
+export const ColorSmoothingParams = {
+    smoothColors: PD.MappedStatic('auto', {
+        auto: PD.Group({}),
+        on: PD.Group({
+            resolutionFactor: PD.Numeric(2, { min: 0.5, max: 6, step: 0.1 }),
+            sampleStride: PD.Numeric(3, { min: 1, max: 12, step: 1 }),
+        }),
+        off: PD.Group({})
+    }),
+};
+export type ColorSmoothingParams = typeof ColorSmoothingParams
+
+export function hasColorSmoothingProp(props: PD.Values<any>): props is PD.Values<ColorSmoothingParams> {
+    return !!props.smoothColors;
+}
+
+export function getColorSmoothingProps(smoothColors: PD.Values<ColorSmoothingParams>['smoothColors'], preferSmoothing?: boolean, resolution?: number) {
+    if ((smoothColors.name === 'on' || (smoothColors.name === 'auto' && preferSmoothing)) && resolution && resolution < 3) {
```

Comment:
What's the purpose of the ``resolution < 3`` check? Maybe make this a ``maxResolution`` option in ``auto`` and ``on``?

Reply:
The smoothing uses vertex positions as sample points. At higher resolutions (of e.g. coarse gaussian surfaces) there are only a few vertices per color. In that case smoothing isn't helpful as the linear interpolation in the shader is enough. It is a bit of a magic number but until we see a problem with it I think we can keep it.

Reply:
Okay, makes sense.